checking for libzip... not found

I get this error, trying to install it on ubuntu 18.04:
`checking for libzip... not found`
`configure: error: Please reinstall the libzip distribution`
`ERROR: Service 'php' failed to build: The command '/bin/sh -c buildDeps="         default-libmysqlclient-dev         libbz2-dev         libsasl2-dev     "     runtimeDeps="         curl         git         libfreetype6-dev         libicu-dev         libjpeg-dev         libldap2-dev         libpng-dev         libpq-dev         libxml2-dev     "     && apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y $buildDeps $runtimeDeps     && docker-php-ext-install bcmath bz2 calendar iconv intl mbstring mysqli opcache pdo_mysql soap zip     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/     && docker-php-ext-install gd     && pecl install redis     && docker-php-ext-enable redis.so     && apt-get purge -y --auto-remove $buildDeps     && rm -r /var/lib/apt/lists/*' returned a non-zero code: 1
Makefile:27: recipe for target 'install' failed
make: *** [install] Error 1`

--- a/.docker/php-fpm/Dockerfile
+++ b/.docker/php-fpm/Dockerfile
@@ -6,6 +6,7 @@ RUN buildDeps=" \
         default-libmysqlclient-dev \
         libbz2-dev \
         libsasl2-dev \
+	libzip-dev \
     " \
     runtimeDeps=" \
         curl \

